static/thing.js

```diff
@@ -0,0 +1,21 @@
+class Thing {
+    constructor(fill = 'rgba(0,255,0,.6)') {
+        this.keyframes = []; // List of boxes corresponding to keyframes
+        this.fill = fill
```

Comment:
Does every thing have a unique color? Like is Car A's color different from Car B? Currently, I was just going to use a single type of color for all types of an object. Like all cars would have a red border while all trucks would have a blue border.